pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
+> More information: <https://developer.gnome.org/dconf>.
+
+- Write a string to a dconf path (note the nested quotes):
```

Comment:
is that last bit needed?

Reply:
I can remove it if you think I should, I figured I'd point out that the *value* of the string needs to be wrapped in single quotes.

Reply:
I think it's safer to keep it, since it's easy to overlook:
![image](https://user-images.githubusercontent.com/50295997/118356429-7abdeb00-b575-11eb-992b-22e177ce28c0.png)

Reply:
Also, why is the single quote nested in the double one, but in array example it's the other way around?

Reply:
I thinks it's fine to keep it too.

Reply:
Ok, however I don't think it makes sense

What do you think about 
```suggestion
- Write a string to a dconf path (within the nested quotes):
```

Reply:
I don't really see how that adds any value 🤔

Reply:
It makes more sense, in my opinion,

Reply:
With `note` it sounds like a warning and it can be read as a seperate sentence. With `within` in sounds like a part of the sentence before it and will likely be read as such.

Reply:
> Also, why is the single quote nested in the double one, but in array example it's the other way around?

Good question, it doesn't matter in this example but I'll edit the array example to match :+1:

Reply:
push this commit then edit the array

Reply:
> push this commit then edit the array

Push what commit?

But yeah, I've already edited the array example

Reply:
The suggested change above

Reply:
I agree with @bl-ue, I don't think the suggested change is more clear.

I also agree with @marchersimon, but I'm not sure whether they're advocating for or against the suggestion :thinking:

Reply:
I'd prefer to keep it how it is (`note`).